other/materials_designer/create_grain_boundary_film.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>Why is index set to 1 - should be 0</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
An artifact from when I was testing that